### PR TITLE
[Protobuf] simplify sort expression

### PR DESCRIPTION
**Pull request by gaurav1086**
**What type of PR is this?**
> Uncomment only one ` /kind <>` line, hit enter to put that in a new line, and remove leading whitespace from that line:
>
> /kind api-change
> /kind bug
/kind cleanup
> /kind deprecation
> /kind design
> /kind documentation
> /kind failing-test
> /kind feature
> /kind flake

**What this PR does / why we need it**:
use sort.Strings(...) instead of sort.Sort(sort.StringSlice(...))
**Which issue(s) this PR fixes**:

Fixes #
N/A
**Special notes for your reviewer**:
N/A
**Does this PR introduce a user-facing change?**:

```release-note
NONE
```

**Additional documentation e.g., KEPs (Kubernetes Enhancement Proposals), usage docs, etc.**:

```docs
NONE
```

### CODE REVIEW

File: staging/src/k8s.io/code-generator/cmd/go-to-protobuf/protobuf/generator.go [hunk 1]
```diff
@@ -399,7 +399,7 @@ func (b bodyGen) doStruct(sw *generator.SnippetWriter) error {
 				}
 				extras = append(extras, fmt.Sprintf("%s = %s", k, v))
 			}
-			sort.Sort(sort.StringSlice(extras))
+			sort.Strings(extras))
```
Comment (wojtek-t) [inline staging/src/k8s.io/code-generator/cmd/go-to-protobuf/protobuf/generator.go:402]: There is no point in retesting - this line doesn't compile (too many ")" at the end.

Comment (gaurav1086) [reply staging/src/k8s.io/code-generator/cmd/go-to-protobuf/protobuf/generator.go]: My bad, thanks for pointing out. Changes done.

**Pull request merged**